port/zos390/omrsl.c

```diff
@@ -287,6 +287,12 @@ omrsl_lookup_name(struct OMRPortLibrary *portLibrary, uintptr_t descriptor, char
 	{
 		handle = (dllhandle *)descriptor;
 		address = (void *)dllqueryfn(handle, name);
+		JavaVersion jdk17 = JavaVersion.version("17");
```

Comment:
You dont need to add the version check. Other platforms (ie. Linux) already search for functions and variables so doing the same here makes things consistent

Reply:
so if `dllqueryfn` returns NULL then call `dllqueryvar`